clarifai/models/api.py

```diff
@@ -1,283 +0,0 @@
-# Copyright 2023 Clarifai, Inc.
```

Comment:
i don't think this file or the __init__.py was supposed to be deleted.

Reply:
Added this one back